quality.gradle

```diff
@@ -78,11 +78,13 @@ task pmd(type: Pmd, group: 'verification') {
     }
 }
 
-android {
-    lintOptions {
-        abortOnError true
-        xmlReport true
-        htmlReport true
-        lintConfig file("${project.rootDir}/AndroidCodeQualityConfig/quality/lint/lint.xml")
+afterEvaluate {
```

Comment:
That's Weird. If I doesn't wrap the android closure with afterEvalate, I will run somthing like
```gradle
* What went wrong:
A problem occurred evaluating script.
> Could not find method android() for arguments [quality_ee33f2rl273pgcfceiuwdfama$_run_closure7@6094e927] on project ':commonLibrary' of type org.gradle.api.Project.
```
I am using:
Gradle: 3.1.0
Gradle Android Plugin: 2.2.2
Java: java version "1.8.0_102"
OS: macOS 10.12.1